specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -392,6 +392,161 @@
+            "schema": {
+              "$ref": "#/definitions/errorResponseBody"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/SupportPlan/default": {
+      "get": {
+        "operationId": "SupportPlanDefault_Get",
+        "x-ms-examples": {
+          "SupportPlan": {
+            "$ref": "./examples/supportPlanDefault.json"
+          }
+        },
+        "description": "returns the support plan for given subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/DefaultSupportPlanResponseResult"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/errorResponseBody"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/Supportplan/default": {
+      "put": {
+        "operationId": "PurchaseSupportPlan_CreateOrUpdate",
+        "x-ms-examples": {
+          "PurchaseSupport": {
+            "$ref": "./examples/purchaseSupportPlan.json"
+          }
+        },
+        "description": "The operation to purchase/convert support plan.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/supportPlanNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query.",
+            "schema": {
+              "$ref": "#/definitions/SupportPlan"
+            }
+          },
+          "202": {
+            "description": "Accepted. Purchase support plan is in progress.",
+            "headers": {
+              "Location": {
+                "description": "GET this URL to retrieve the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/errorResponseBody"
```

Comment:
nit: It would be best to refer to the standard error definition under "specifications/common-types/resource-management/v1/types.json#/definitions/ErrorResponse". With this, PSH will be able to correctly extract the message and display to the user. 

Note: this may be an SDK breaking change. If this is indeed a recent preview version, we encourage you to update this version to refer to that file. If it is stable version, please make a note and fix in future version.